optimade/server/entry_collections.py

```diff
@@ -16,7 +16,7 @@
 
 try:
     ci_force_mongo = bool(int(os.environ.get("OPTIMADE_CI_FORCE_MONGO", 0)))
-except Exception:
+except TypeError:
```

Comment:
```suggestion
except ValueError:
```

Unless you have any examples where it raises a `TypeError`?

Reply:
`int(None)`

Reply:
I guess we should catch both?